userguide/tutorials/getting_started.adoc

```diff
@@ -136,47 +136,55 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
-==== Kill Bill
+==== Kill Bill Configuration
 
-The Kill Bill configuration properties need to be added at the end of the file *CATALINA_HOME/conf/catalina.properties*. The minimal set of system properties required is the ones for the jdbc parameters (refer to our http://docs.killbill.io/latest/userguide_configuration.html[configuration guide] for more details). For example (update the values as needed):
+. Add the Kill Bill configuration properties at the end of the *CATALINA_HOME/conf/catalina.properties* file.  At the  minimum, the  JDBC  parameters need to be added. So, for example, you can add the following(update the values as needed):
+
+ org.killbill.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
+ org.killbill.dao.user=killbill
+ org.killbill.dao.password=killbill
+ org.killbill.billing.osgi.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
+ org.killbill.billing.osgi.dao.user=killbill
+ org.killbill.billing.osgi.dao.password=killbill
+ logback.configurationFile=/var/lib/killbill/config/logback.xml
 
-[source,java]
-----
-org.killbill.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
-org.killbill.dao.user=killbill
-org.killbill.dao.password=killbill
-org.killbill.billing.osgi.dao.url=jdbc:mysql://127.0.0.1:3306/killbill
-org.killbill.billing.osgi.dao.user=killbill
-org.killbill.billing.osgi.dao.password=killbill
-logback.configurationFile=/var/lib/killbill/config/logback.xml
-----
 
-Place the logging configuration file https://github.com/killbill/killbill-cloud/blob/master/ansible/templates/killbill/logback.xml.j2[logback.xml] as `/var/lib/killbill/config/logback.xml` (refer to the Jinja templating language instructions above).
+. Place the logging configuration file https://github.com/killbill/killbill-cloud/blob/master/ansible/templates/killbill/logback.xml.j2[logback.xml] as `/var/lib/killbill/config/logback.xml` (refer to the Jinja templating language instructions above).
 
-Download the current stable version of the Kill Bill war from https://search.maven.org/search?q=a:killbill-profiles-killbill[Maven Central] (make sure to download the `killbill-profiles-killbill-X.Y.Z.war` file and not the `jetty-console.war`, `jar-with-dependencies.war` or `jar-with-dependencies-sources.war` one) and place it under *CATALINA_HOME/webapps/ROOT.war* (delete the directory *CATALINA_HOME/webapps/ROOT* if it exists).
+. Download the current stable version of the Kill Bill war from https://search.maven.org/search?q=a:killbill-profiles-killbill[Maven Central] (make sure to download the `killbill-profiles-killbill-X.Y.Z.war` file and not the `jetty-console.war`, `jar-with-dependencies.war` or `jar-with-dependencies-sources.war` one) and place it under *CATALINA_HOME/webapps/ROOT.war* (delete the directory *CATALINA_HOME/webapps/ROOT* if it exists).
 
 Once this is setup, you can start the Tomcat container using the script *CATALINA_HOME/bin/startup.sh*. The server will be started in the background but logs can be followed at *CATALINA_HOME/logs/catalina.out*.
 
```

Comment:
It would be nice in the end to summarize the installation with as view of the filesystem:

(My formatting is not great, just to show the idea)

```
$CATALINA_HOME -
                                - config/
                                          - ...
                                 - logs/
                                         - catalina.out  
                                         - killbill.out  
                                ...
```

Reply:
I've now added installation summaries where possible